vyxal/parse.py

```diff
@@ -331,32 +249,23 @@ def parse(tokens: list[lexer.Token]) -> list[Structure]:
 
             remaining = parse(tokens)
             if head.value == "⁽":
-                structures.append(
-                    Structure(StructureType.LAMBDA, ["1", [[remaining[0]]]])
-                )
+                structures.append(structure.Lambda(1, [[remaining[0]]]))
             else:
                 structures.append(
-                    Structure(
-                        StructureType.MONADIC_MODIFIER,
-                        [head.value, [remaining[0]]],
-                    )
+                    structure.MonadicModifier(head.value, remaining[0])
                 )
             structures += remaining[1:]
             break
         elif head.value in DYADIC_MODIFIERS:
             remaining = parse(tokens)
             if head.value == "‡":
```

Comment:
add a comment to describe what this is doing so I don't have to go and look up the meaning of `‡`?